client/lib/abtest/active-tests.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
 	personalPlan: {
-		datestamp: '20160622',
+		datestamp: '20160623', //was 20160622
```

Comment:
I don't think we need this comment, but if you want something here it should probably be more descriptive. `A previous test run for a short period on 20160622`